internal/querynode/flow_graph_delete_node.go

```diff
@@ -51,6 +51,7 @@ func (dNode *deleteNode) Operate(in []flowgraph.Msg) []flowgraph.Msg {
 	// 1. filter segment by bloom filter
 	for _, delMsg := range dMsg.deleteMessages {
 		if dNode.replica.getSegmentNum() != 0 {
+			log.Debug("delete in historical replica")
```

Comment:
Add more log info is recommended, such as `collectionID`,  `collectionName` and `timestamp`.